tedana/workflows/tedana.py

```diff
@@ -264,12 +263,21 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
         os.mkdir(out_dir)
 
     if debug and not quiet:
+        # ensure old logs aren't over-written
+        logname = op.join(
+                out_dir,
+                'tedana_run')
+
+        while op.isfile(logname):
+            logname += '_old'
```

Comment:
This is clever, but I think it will not do what we want. The newest log file will end up being `tedana_run_old_old_old.txt` (or whatever). It would take a bit more effort, but I think a preferable approach would be to identify all of the run files and to rename each existing file with an additional `_old` in the name. So
`tedana_run.txt` --> `tedana_run_old.txt`
`tedana_run_old.txt` --> `tedana_run_old_old.txt`
and the new file is `tedana_run.txt`

That said, I don't know if it's worth the effort, since the naming for the log file will almost certainly change to match the BIDS Derivatives specification. I have no clue what that file will end up looking like.

I would recommend trying to see if you can implement the above fairly easily. If you can't, then maybe only retain two (`tedana_run.txt` as the new one and `tedana_run_old.txt` for any existing one).

Reply:
Well... it's hacky, but I could change old->new... I think if people are actually going to re-run tedana on the same folder over and over we may want something generally more sophisticated, but I figured that this pass would at least make it possible to work on integrating `--debug` to CircleCI

Reply:
After further reflection, it's not that bad to incorporate this suggestion. Let me fiddle and test before pushing a corrective commit.

Reply:
This should be fixed in 1b3c52c